src/components/ContentHighlights/HighlightStepper/HighlightStepperTitle.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import {
+  Stack, Col, Form, Image, Container,
+} from '@edx/paragon';
+import pencil from '../data/images/pencil-circle.svg';
+import { STEPPER_STEP_TEXT } from '../data/constants';
+
+const HighlightStepperTitle = () => (
+  <Container size="md">
+    <Stack>
+      <Col>
+        <Stack className="mb-3" direction="horizontal">
+          <Image className="pgn__icon" src={pencil} alt="" />
```

Comment:
I think we should use an icon that's already part of [Paragon](https://paragon-openedx.netlify.app/foundations/icons). I believe this icon pencil-circle is from `font-awesome`, which is something we are generally moving away from.